src/pybind/cephfs/cephfs.pyx

```diff
@@ -500,11 +504,9 @@ cdef class LibCephFS(object):
             raise Error("libcephfs_initialize failed with error code: %d" % ret)
 
         self.state = "configuring"
-        if conffile is not None:
-            # read the default conf file when '' is given
-            if conffile == '':
-                conffile = None
-            self.conf_read_file(conffile)
+        # skip reading default conf
+        if conffile == False or conf == False:
```

Comment:
I'm confused, this looks like you *are* reading the conf file when False, which is the opposite of the documentation above?

Reply:
Yeah, I got a little confused looking at the code in master; fixed now. Thanks!